index.html

```diff
@@ -8,7 +8,7 @@
     <section class="grid-middle hero">
         <div class="col-5_md-12">
             <h1>Open-Source,<br />Cloud-Native Storage for Kubernetes</h1>
-            <p>Production ready File, Block and Object Storage</p>
+            <p>Production ready management for File, Block and Object Storage</p>
```

Comment:
"orchestration"?
"orchestration and management"?
just thoughts, not required for this PR

Reply:
I was trying to keep it as brief as possible for this one-liner. Orchestration also crossed my mind, but I would think "management" is more understood at this level than "orchestration"